Formula/lesspipe.rb

```diff
@@ -33,4 +33,12 @@ def install
     assert File.exist?("homebrew.tar.gz")
     assert_match /file2.txt/, shell_output("tar tvzf homebrew.tar.gz | #{bin}/tarcolor")
   end
+  
+  def caveats
+    <<-EOS
+      lesspipe.sh gets symlinked to /usr/local/bin/lesspipe.sh.
+      Append the following to your .bash_profile/.bashrc:
+      export LESSOPEN="|/usr/local/bin/lesspipe.sh %s" LESS_ADVANCED_PREPROCESSOR=1
```

Comment:
- The first line is not really useful
- Are the instructions Homebrew-specific?
- Please do not hardcode `/usr/local/bin` but use `HOMEBREW_PREFIX` in caveats